src/modules/flow/http-client/http-client.c

```diff
@@ -476,6 +477,8 @@ boolean_process_data(struct sol_flow_node *node,
 {
     bool result;
 
+    SOL_HTTP_RESPONSE_CHECK_API(response, -EINVAL);
+
```

Comment:
Actually, this is called by generic_request_finished(), which handles the http response. However the function generic_request_finished() already checks the response's api version. I'm removing it.